### PR TITLE
DOC-2782 Tech Writer: Edited 0.12 changelog.

### PR DESCRIPTION
**Pull request by SPPWilliams**
#### Description
Describe your changes here.

Tech Writer edit for 0.12 changelog.

### CODE REVIEW

**Review by MatthewSandfordImprobable — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,65 +31,63 @@ These functions and structs can be referenced in both code and blueprints it may
...
   1. The time elapsed since the last sent Spatial position update is greater than or equal to `PositionUpdateLowerThresholdSeconds` AND the distance travelled since the last update is greater than or equal to `PositionUpdateLowerThresholdCentimeters`.
   1. The time elapsed since the last sent Spatial position update is greater than or equal to `PositionUpdateThresholdMaxSeconds` AND the Actor has moved a non-zero amount.
   1. The distance travelled since the last Spatial position update was sent is greater than or equal to `PositionUpdateThresholdMaxCentimeters`.
-- New setting "Auto-stop local SpatialOS deployment" that allows you to specific Never (doesn't automatically stop), OnEndPIE (when a PIE session is ended) and OnExitEditor (only when the editor is shutdown). The default is OnExitEditor.
+- New setting "Auto-stop local SpatialOS deployment" allows you to specify Never (doesn't automatically stop), OnEndPIE (when a PIE session is ended) and OnExitEditor (only when the editor is shutdown). The default is OnExitEditor.
```
Comment (improbable-danny) [inline CHANGELOG.md:43]:
```suggestion
- New setting "Auto-stop local SpatialOS deployment" allows you to specify `Never` (doesn't automatically stop), `OnEndPIE` (when a PIE session is ended) and `OnExitEditor` (only when the editor is shutdown). The default is `OnExitEditor`.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,14 +14,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Breaking changes:
```
Comment (joshuahuburn) [inline CHANGELOG.md:14]: While we're here we should probably update the date just above :D It's currently `2020-12-14`

**Review by improbable-danny — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -31,65 +31,63 @@ These functions and structs can be referenced in both code and blueprints it may
...
   - Runtime and Inspector binary fetching is now handled by the GDK. To fetch these binaries you must start the game project at least once.
-  - The Runtime and Inspector will be restarted between each editor session.
+  - The Runtime and Inspector will be restarted between each Editor session.
   - Support for standalone launch configurations introduced, these are now generated by default for your local deployments, worker configurations for local deployments are now stored inside the launch config. 
     - Generated local launch configs can still be found in `<GameProject>\Game\Intermediate\Improbable\<MapName>_LocalLaunchConfig.json`.
   - Cloud deployments still use the older launch config and worker config format, we still generate these automatically for your maps.
     - Generated cloud launch configs can  be found in `<GameProject>\Game\Intermediate\Improbable\<MapName>_CloudLaunchConfig.json`.
   - The Launch Configuration Generator tool can be used to generate either local or cloud launch configs via a checkbox in the GUI.
-  - Spot and SpatialD dependencies have been removed.
```
Comment (joshuahuburn) [inline CHANGELOG.md:81]: How come this line has been removed?

Comment (SPPWilliams) [reply CHANGELOG.md:81]: It's a duplicate. Line 30 also includes this:
- Spot and SpatialD (Spatial Service) dependencies have been removed.

File: CHANGELOG.md (hunk 2 above)
Comment (SPPWilliams) [reply CHANGELOG.md:14]: Good spot! I've submitted a second request to fix this

**Pull request merged**